source/fundamentals/crud/write-operations/bulk.txt

```diff
@@ -288,18 +311,200 @@ operations to execute in any order:
       { "_id": 6, "name": "Zaynab Omar", "age": 37 }
 
 For more information about the methods and classes mentioned in this section,
-see the following API Documentation:
+see the following API documentation:
 
 - `BulkWriteOptions <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/BulkWriteOptions.html>`__
 - `ordered() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/BulkWriteOptions.html#ordered(boolean)>`__
 
+.. _java-sync-client-bulk-write:
+
+Client Bulk Write
+-----------------
+
+When connecting to a deployment running {+mdb-server+} 8.0 or later,
+you can use the ``MongoClient.bulkWrite()`` method to write
+to multiple databases and collections in the same cluster. The ``MongoClient.bulkWrite()``
+method does not split different kinds of operations into different batches.
+
+In the examples in preceding sections of this page, the ``MongoCollection.bulkWrite()`` method 
+takes a list of ``WriteModel`` instances in which the specified subclass of 
+``WriteModel`` represents the corresponding write operation. For example, an
+instance of ``InsertOneModel`` represents an operation to insert one document. 
+
+Similarly, the ``MongoClient.bulkWrite()`` method takes a 
+list of ``ClientNamespacedWriteModel`` instances to represent different write operations.
+For example, an instance of ``ClientNamespacedInsertOneModel`` represents an
+operation to insert one document.
+
+You can construct instances of ``ClientNamespacedWriteModel`` using the following
+methods:
+
+- ``insertOne()``
+
+- ``updateOne()``
+
+- ``updateMany()``
+
+- ``replaceOne()``
+
+- ``deleteOne()``
+
+- ``deleteMany()``
+
+These methods are used to construct corresponding write models. 
+For example, ``ClientNamespacedWriteModel.updateOne()`` is used to
+construct a ``ClientNamespacedUpdateOneModel`` instance, which represents an
+update operation.
+
+These methods take a ``MongoNamespace`` object that defines which
+database and collection to write to. Some, such as ``insertOne()``, can take a 
+``Document`` instance that defines information about the write operation. 
+Some other methods, such as ``updateOne()`` and ``replaceOne()``, take a filter 
+object that defines the subset of documents to be updated or replaced.
```

Comment:
for comprehensiveness and clarity, we probably want to specify which methods take what. you could give each method its own subsection, or add a parameter list/table for each one